backend/logic/courses/courses.js

```diff
@@ -145,9 +147,27 @@ var addCourseOfferings = async courseOfferings => {
   }
 };
 
+let phaseOutCourse = async courseCode => {
+  let connection = await mysql.getNewConnection();
+  try {
+    await format.verifyCourse(courseCode);
+    await connection.query(
+      "UPDATE courses SET phased_out = TRUE WHERE course_code = ?",
```

Comment:
Not 100% sure, but from stevens explanation from the slack 

> @channel I don't think I explained myself clear during our meeting for beginning a transaction in MySQL. There is absolutely no need to begin and commit a transaction unless if you're writing data into the database (UPDATE, DELETE, INSERT, etc.). By default, MySQL database have auto commits turned on, but, in some cases, you want to temporarily disable auto commits until you're sure a block of queries will execute successfully which is where transactions come in. A transaction is used to preserve atomicity on multiple queries meaning that if one of the queries fails then the entire transaction must be rollback 

TLDR; If you're working with SELECTs only, you don't need to begin a transaction since it prevents legitimate write instructions from accessing the table

Do you have to use the transaction and commit function?

Reply:
A race condition is possible where connection A removes data before connection B updates it but this a likely extremely rare race condition considering how often courses get phased out (maybe once every 5 years) and either way if the call fails there's no net impact on the database so it's more for styling points lol